### PR TITLE
Adjust formatting to match GitHub web editor

### PR DESCRIPTION
**Pull request by rob-luke**
This PR was created by opening the file in the GitHub editor. Removing the extra space on line 869. GitHub then auto formatted the table.

Using the default GitHub formatting will mean people can make quick changes in the web editor, rather than having to clone and edit in local editor then push up.

### CODE REVIEW

**Comment by fangq** [comment]
I am not sure what was the issue this patch is trying to address - does github editor add something in extra? what happens if you use any other editors? if other text editor does not have this issue, I suggest this is not a spec issue, but a github editor's issue.

**Comment by rob-luke** [comment]
This is absolutely a GitHub editor issue. But this repository is hosted on GitHub. 
And most contributors including myself want to use the web interface.

And every time I (or any contributor) wants to suggest a change it automatically reformats the table in this way. This means that it's continuously going to happen in unrelated PRs.

Also the table brackets were misaligned so I fixed that.

Is there a negative to implementing this change?

**Comment by fangq** [comment]
@rob-luke, I see what you meant by Github editor - you mean the web-based editor, not github desktop editor.

however, I don't see a space on line https://github.com/fNIRS/snirf/blame/master/snirf_specification.md#L869

if the line number was shifted, can you update? just want to reproduce the issue myself first.

File: snirf_specification.md [hunk 1]
```diff
@@ -866,7 +866,7 @@ Chunked data is allowed to support real-time data streaming
 * **Type**:  numeric
 * **Location**: `/nirs(i)/aux(j)/timeOffset`
 
-This variable specifies the offset of the file time origin relative to absolute
```
Comment (rob-luke) [inline snirf_specification.md:869]: This space. Was just a token change. But I don't like hanging white spaces 😉

**Comment by rob-luke** [comment]
> Github editor - you mean the web-based editor, not github desktop editor.

I completely forgot there was a desktop version. My bad. I almost exclusively use the web editor for single text changes like this.

I think to be as accessible to the community as possible it's good to enable the tools people are most likely to use. And for a project like this (all text, no code) I assume that's the web editor.

If you can't reproduce then we can try together on today's call.

**Comment by fangq** [comment]
@rob-luke, just sent you a message, can meet now if you have time.

**Comment by fangq** [comment]
@rob-luke, this is what happened: here is the hex code for a matching line before and after the change
github_text_editor_table_reformatting

the first half is the original text, and the bottom half (selected) is from the github web editor

you can see the change was the replacement of U+2000 EN_QUAD (in UTF8: 0xE2 80 80) to U+2002 (EN_SPACE) (in UTF8: 0xE2 80 82)

https://www.utf8-chartable.de/unicode-utf8-table.pl?start=8192&number=128

although based on [wikipedia](https://en.wikipedia.org/wiki/Quad_(typography)#:~:text=An%20en%20quad%20is%20a,U%2B2003%20EM%20SPACE%20respectively), these two are "canonically equivalent", somehow, github recognizes that this is a table and used EN_SPACE to reformat the table for alignment (there are about 4 lines in the original table was not entirely aligned in a text editor, like vi). 

again, I wouldn't say this is an issue of the spec file itself, but agree that given github web editor's preference to EN_SPACE, I am ok to switch. 

please figure out how to resolve the conflict marked above, and I will accept.

**Comment by rob-luke** [comment]
> again, I wouldn't say this is an issue of the spec file itself, but agree that given github web editor's preference 

Thanks for agreeing to this. While this isn't a spec issue, changing this will make it easier for me to help maintain this project.

@fangq this is ready for another review. Cheers.

**Comment by rob-luke** [comment]
Thanks @fangq

**Pull request merged**